packages/chart/src/components/EditorPanel/EditorPanel.tsx

```diff
@@ -1958,7 +1958,7 @@ const EditorPanel = () => {
                         }
                         updateField={updateField}
                       />
-                      {false && visHasBrushChart && (
+                      {true && visHasBrushChart && (
```

Comment:
If we aren't hiding the brush anymore lets remove the true statement here.

Reply:
Fixed!